### PR TITLE
Feat: Add Contact Us Section

### PR DESCRIPTION
**Pull request by tharshita**
# Notion Task

https://www.notion.so/Contact-Us-option-2513e86d53774960bbb4f203578a4fdd
https://www.notion.so/Hyperlink-to-CareerSocius-Website-4d39818218b346a58f142ff1d8d2d8ed

# Changes Made In This PR

- added hyperlink to website
- added contact us with mailto career socius email

# Screenshots

_for UI changes, show a before and after shot_\
**Before:**
Screenshot 2022-03-25 at 9 16 54 PM

**After:**
Screenshot 2022-03-26 at 3 09 05 PM

Screenshot 2022-03-19 at 5 09 42 PM

### CODE REVIEW

**Review by millasml — CHANGES_REQUESTED** [review]
looking good! can we make it such that

- the "About" header is gone?
- the resume builder version tag thing is at the bottom most
- the contact us section is styled exactly the same as the review section

File: src/constants/leftTabBarSections.js [hunk 1]
```diff
@@ -90,4 +90,7 @@ export const checkSections = [
   {
     id: 'about',
   },
+  {
+    id: 'contact-us',
```
Comment (millasml) [inline src/constants/leftTabBarSections.js:94]: lets make this `contactUs`

**Review by millasml — CHANGES_REQUESTED** [review]
close!!

File: src/constants/leftTabBarSections.js (hunk 1 above)
Comment (tharshita) [reply src/constants/leftTabBarSections.js]: done!

**Review by millasml — APPROVED** [review]

**Pull request merged**